physics/CONV/SAMF/samfdeepcnv.meta

```diff
@@ -188,6 +188,14 @@
   type = real
   kind = kind_phys
   intent = in
+[cscale]
+  standard_name = scaling_conv_cloud_water_in_sas
```

Comment:
There are many existing standard names that use "multiplicative_tuning_parameter_for_X". I would change to the following to be consistent:
`multiplicative_tuning_parameter_for_convective_cloud_water`